tests/testthat/run-tests.R

```diff
@@ -0,0 +1,2 @@
+library(devtools)
+devtools::test()
```

Comment:
add new line to end of file

```suggestion
devtools::test()

```